eth/api.go

```diff
@@ -1506,17 +1508,58 @@ func NewPrivateDebugAPI(eth *Ethereum) *PrivateDebugAPI {
 	return &PrivateDebugAPI{eth: eth}
 }
 
-// ProcessBlock reprocesses an already owned block.
-func (api *PrivateDebugAPI) ProcessBlock(number uint64) (bool, error) {
+// Block processes the given block's RLP but does not import the block in to
+// the chain.
+func (api *PrivateDebugAPI) Block(blockRlp []byte, logger VmLoggerOptions) (bool, error) {
```

Comment:
Agree, though `ProcessBlock` is quite a shitty name. `debug.TraceBlock` is something maybe